pkg/go.mod

```diff
@@ -61,3 +62,5 @@ require (
 	sourcegraph.com/sourcegraph/appdash v0.0.0-20190731080439-ebfcffb1b5c0
 	sourcegraph.com/sourcegraph/appdash-data v0.0.0-20151005221446-73f23eafcf67 // indirect
 )
+
+replace github.com/yuin/goldmark => github.com/pgavlin/goldmark v1.1.33-0.20200616164752-aaa9ff3e1d61
```

Comment:
This is breaking the downstream testing. I'm going to move my fork into its own module to avoid this.